This allows us to get some function from math.h that other platforms are
getting elsewhere.
Can potentially send upstream with ifdef Solaris statement.

diff --git a/dom/media/webaudio/ThreeDPoint.h b/dom/media/webaudio/ThreeDPoint.h
--- a/dom/media/webaudio/ThreeDPoint.h
+++ b/dom/media/webaudio/ThreeDPoint.h
@@ -8,6 +8,7 @@
 #define ThreeDPoint_h_
  
 #include <cmath>
+#include <math.h>
 #include <algorithm>
 
 namespace mozilla {
diff --git a/dom/media/webaudio/blink/DenormalDisabler.h b/dom/media/webaudio/blink/DenormalDisabler.h        
--- a/dom/media/webaudio/blink/DenormalDisabler.h
+++ b/dom/media/webaudio/blink/DenormalDisabler.h
@@ -27,6 +27,7 @@
 
 #define _USE_MATH_DEFINES
 #include <cmath>
+#include <math.h>
 #include <float.h>
 
 namespace WebCore {
